kadena/docs/src/design/longest_chain.md

```diff
@@ -0,0 +1,45 @@
+# Longest chain proof
+
+The light client needs to keep track of the longest chain to ensure that it is in 
+sync with the running chain. The longest chain is the chain with the most accumulated
+work.
+
+There are a few predicates that need to be satisfied to ensure that the light client
+is in sync with the longest chain:
+1. The newly presented list of blocks supposed to represent the tip
+    of the longest chain must be mined on top of an already verified layer block.
+2. The height of each layer block must be strictly increasing.
+3. The parent hash of each inner chain block headers must match the hash of the
+    previous block of the given chain ID.
+4. The work produced for a given chain block header must be inferior or equal
```

Comment:
Just double checking (might be a confusion on my part): Is this "inferior or equal" correct here, or should it be "greater or equal"?

Reply:
For a given block header we want the PoW hash to be inferior or equal to the target difficulty. See https://github.com/argumentcomputer/zk-light-clients/blob/dev/kadena/core/src/types/header/chain.rs#L313-L320

Reply:
Ah ok, I see what it means now. The amount of work (in PoW terms) is higher by limiting the hash value to be smaller than the target. That is, only the hash value is less than the target but the amount of work that hash represents is still greater than the difficulty target (i.e. the hash target is its own thing derived from the current difficulty)